apps/probe/src/components/probe-list/probe-list.tsx

```diff
@@ -23,6 +24,18 @@ import {
 } from "../../hooks/use-probes";
 import styles from "./probe-list.module.css";
 
+const getOsVariant = (dos: string) => {
```

Comment:
You can use the `Platform` type here it should come from the SDK

```suggestion
const getOsVariant = (dos: Platform) => {
```